huaweicloud/services/acceptance/dcs/resource_huaweicloud_dcs_redis_instance_test.go

```diff
@@ -0,0 +1,520 @@
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&redisInstance,
+		getResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccResourceDcsRedisInstance_basic(
+					map[string]string{
+						"name":       name,
+						"version":    "5.0",
+						"capacity":   "4",
+						"specCode":   "redis.proxy.xu1.large.4",
+						"chargeMode": "prePaid",
+						"periodUnit": "month",
+						"autoRenew":  "false",
+						"period":     "1",
+					},
+				),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+					resource.TestCheckResourceAttr(resourceName, "engine_version", "5.0"),
+					resource.TestCheckResourceAttr(resourceName, "capacity", "4"),
+					resource.TestCheckResourceAttr(resourceName, "resource_spec_code", "redis.proxy.xu1.large.4"),
+					acceptance.TestCheckResourceAttrWithVariable(resourceName, "available_zones.0",
+						"${data.huaweicloud_availability_zones.zones.names.0}"),
+					resource.TestCheckResourceAttr(resourceName, "port", "6380"),
+					resource.TestCheckResourceAttr(resourceName, "maintain_begin", "22:00:00"),
+					resource.TestCheckResourceAttr(resourceName, "maintain_end", "02:00:00"),
+					resource.TestCheckResourceAttr(resourceName, "backup_policy.0.backup_at.#", "3"),
+					resource.TestCheckResourceAttr(resourceName, "backup_policy.0.backup_type", "auto"),
+					resource.TestCheckResourceAttr(resourceName, "backup_policy.0.begin_at", "00:00-02:00"),
+					resource.TestCheckResourceAttr(resourceName, "backup_policy.0.timezone_offset", "+0700"),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "2"),
+					resource.TestCheckResourceAttr(resourceName, "tags.level", "A"),
+					resource.TestCheckResourceAttr(resourceName, "whitelists.#", "2"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccResourceDcsRedisInstance_rwsplit(t *testing.T) {
+	var redisInstance instances.DcsInstance
+	resourceName := "huaweicloud_dcs_redis_instance.instance_1"
+	name := acceptance.RandomAccResourceName()
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&redisInstance,
+		getResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccResourceDcsRedisInstance_basic(
+					map[string]string{
+						"name":       name,
+						"version":    "5.0",
+						"capacity":   "8",
+						"specCode":   "redis.ha.xu1.large.p2.8",
+						"chargeMode": "prePaid",
+						"periodUnit": "month",
+						"autoRenew":  "false",
+						"period":     "1",
```

Comment:
chargeMode, periodUnit, autoRenew,period are not used in testAccResourceDcsRedisInstance_basic

Reply:
done